docs/src/components/action/NpmCopyButton.tsx

```diff
@@ -16,19 +17,15 @@ const StyledButton = styled(Button)`
   outline: 0;
   border: 0;
   margin: 0;
-  user-select: none;
   vertical-align: middle;
   text-decoration: none;
   text-transform: initial;
   scroll-snap-margin-top: calc(var(--MuiDocs-header-height) + 32px);
   scroll-margin-top: calc(var(--MuiDocs-header-height) + 32px);
   min-width: 64px;
   transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
-    box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
-    border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
     color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
   box-shadow: none;
-  border-radius: 6px;
   cursor: copy;
   padding: 0;
   width: max-content;
```

Comment:
Below in the style, why are we using `var(--muidocs-palette-primary-main)` without the theme? I think it would be better to pull this from the theme for type safety and to abstract away the CSS variable value.

Reply:
`Menlo, Consolas, 'Droid Sans Mono', monospace` I think shouldn't be hard coded but `theme.fontFamily.fontFamilyCode`

Reply:
This button seems to be using pure CSS (hence the kebab-case) ⎯ how would I use anything like `theme.palette.primary.main` here? I guess I should convert this to CSS in JS in some way, right?

Reply:
```diff
diff --git a/docs/src/components/action/NpmCopyButton.tsx b/docs/src/components/action/NpmCopyButton.tsx
index 2511fac8ff..c01a9a9061 100644
--- a/docs/src/components/action/NpmCopyButton.tsx
+++ b/docs/src/components/action/NpmCopyButton.tsx
@@ -6,7 +6,7 @@ import { Theme } from '@mui/material/styles';
 import ContentCopyRounded from '@mui/icons-material/ContentCopyRounded';
 import CheckRounded from '@mui/icons-material/CheckRounded';

-const StyledButton = styled(Button)`
+const StyledButton = styled(Button)(({ theme }) => (`
   display: inline-flex;
   gap: 8px;
   align-items: center;
@@ -31,7 +31,7 @@ const StyledButton = styled(Button)`
   width: max-content;
   background-color: transparent;
   color: var(--muidocs-palette-grey-600);
-  font-family: Menlo, Consolas, 'Droid Sans Mono', monospace;
+  font-family: ${theme.typography.fontFamily.fontFamilyCode};
   font-weight: 400;
   font-size: 0.75rem;
   line-height: 1.5;
@@ -59,14 +59,12 @@ const StyledButton = styled(Button)`
       opacity: 1;
     }
   }
-`;
+`));
```